sonaranalyzer-dotnet/tests/SonarAnalyzer.UnitTest/TestCases/ProvideDeserializationMethodsForOptionalFields.CSharp9.cs

```diff
@@ -0,0 +1,59 @@
+﻿using System;
+// Noncompliant@-1 // FP, the location is wrong for records
```

Comment:
weird, it would be interesting to see why this happens

Reply:
Yes, of course. I've marked the rule as `to implement` to be sure we will take a look on it.